src/app.js

```diff
@@ -1,7 +1,23 @@
-import { User, Product } from './models';
+import * as path from 'path';
 import * as config from './config/config.json';
+import { Dirwatcher } from './dirwatcher';
+import { Importer } from './importer';
+import { Product, User } from './models';
 
 console.log(config.name);
 
 new User();
 new Product();
+
+const DATA_DIR_PATH = path.join(__dirname, 'data');
+const dirwatcher = new Dirwatcher().watch(DATA_DIR_PATH, 1000);
+
+dirwatcher.emitter.on(Dirwatcher.DIR_CHANGE_EVENT, async files => {
```

Comment:
Actually, this logic should be placed inside `Importer` class as it should be the basic functionality of it. Pass `Dirwatcher` as a param to Importrer constructor and move this logic to `Importter.comstructor`, so when creating an instance of importer your class will initially watch the emits of dirwatcher